ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host6/eos-cli-config-gen-documentation.yml

```diff
@@ -0,0 +1,4 @@
+---
+eos_cli_config_gen_documentation:
+  enable: true
+  hide_passwords: false
```

Comment:
would be good to check that this works (i.e. password is visible in doc)-> sadly the aaa-root key is not rendered in doc so we can't see it - can we select another key?

Reply:
updated